testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5336,6 +5336,25 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("array_distinct(null)");
   }
 
+  /** Tests {@code ARRAY_REPEAT} function from Spark. */
+  @Test void testArrayRepeatFunc() {
+    final SqlOperatorFixture f0 = fixture();
+    f0.setFor(SqlLibraryOperators.ARRAY_REPEAT);
+    f0.checkFails("^array_repeat(1, 2)^",
+        "No match found for function signature ARRAY_REPEAT\\(<NUMERIC>, <NUMERIC>\\)", false);
+
+    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.SPARK);
+    f.checkScalar("array_repeat(1, 2)", "[1, 1]",
```

Comment:
If count <= 0, it is easy to understand for error message?

Reply:
i check the spark it will be to zero.